### DISCUSSION TITLE
Socket stalled when trying to connect to wss://relay.walletconnect.com

### DISCUSSION THREAD

**Discussion by hengshanMWC** [wevm/viem#2112] [Question]
This error occurs when a user switches wallet accounts, disconnects the wallet, then connects the wallet, switches accounts again, disconnects the wallet.
<img width="946" alt="image" src="https://github.com/wevm/viem/assets/45714422/b61950fd-6b12-4a25-81a5-1b1d826b5f33">

viem@1.21.1
@wagmi/core@1.4.7
@web3modal/wagmi@3.3.2

**Comment by hengshanMWC** [comment]
<img width="775" alt="image" src="https://github.com/wevm/viem/assets/45714422/60660b3c-39a0-4432-9780-8c4715e0207f">
Then the website will appear with no modal, only a curtain scene, which cannot be closed.

**Comment by jxom** [comment]
This is a WalletConnect issue. Best to raise an issue on their repo instead.

**Reply by hengshanMWC** [reply]
Okay, thank you
